scripts/release.sh

```diff
@@ -74,6 +74,18 @@ create_gh_release() {
     --title "Release $release_tag"
 }
 
+build_docker_image() {
+  docker build -t splunk-otel-instrumentation-java .
+  docker tag splunk-otel-instrumentation-java quay.io/signalfx/splunk-otel-instrumentation-java:$release_tag
+  docker tag splunk-otel-instrumentation-java quay.io/signalfx/splunk-otel-instrumentation-java:latest
+}
+
+publish_docker_image() {
+  docker push quay.io/signalfx/splunk-otel-instrumentation-java:latest
```

Comment:
First, don't you need to authenticate to quay before pushing?
Second, you don't push `$release_tag`, is this intentional?

Reply:
Missed the release tag. I thought all repos are pre-configured for quay access (because of a conversation in Jira) but that doesn't seem to be the case. Will add that.